### DISCUSSION TITLE
Recommended system requirements for dedicated transcription PC

### DISCUSSION THREAD

**Discussion by VBodrov** [kaixxx/noScribe#44] [General]
Thank you so much for your work on NoScribe. I tried to use it to transcribe telephone conversations of our operators and was pleased with the quality of the recording. Despite the fact that errors do occur, the overall quality is sufficient to analyze whether employees followed the negotiation scripts, whether they were friendly, and so on. In any case, analyzing the text, albeit with errors, is much easier than listening to hours of negotiations.

My question is the following. When using transcription on work computers, the process takes quite a bit of time. But work computers are laptops without separate video cards.

Therefore, the idea arose to allocate a separate computer specifically for transcribing negotiations. Are there any recommendations regarding what the computer configuration should look like on which decryption would be most effective? And is it possible to fine-tune NoScribe to maximize the use of hardware?

Thanks again for creating your product!

**Comment by kaixxx** [comment]
Hi
Well, this is a little bit of a moral dilemma for me. I appreciate your kind words about my software. But I’m not sure if I should help you make the surveillance of your call-center-agents even more efficient… 
Anyhow, if you promise to be nice to your employees, here is a hint for you: I’m trying to implement an acceleration via the graphics card over the next couple of weeks. This could bring quite an improvement in speed, but it only works with NVIDIA-Cards that support CUDA (google "NVIDA Cuda supported cards" for more information). Other than that, any current PC with a decent CPU should work.

**Reply by VBodrov** [reply]
Don't worry, it's not a surveillance. They know that the conversations are being recorded, because they turn on the recording themselves. And if they want, they can leave it out, but then we can't help them achieve their KPIs. :)

I look forward to the new version with GPU card support. Thank you

**Comment by Lod3** [comment]
Very much excited for CUDA support as well, we are recommending Noscribe to cultural heritage organisations to transcribe local history interviews and they are using mostly old laptops with weak CPU's. Recommending a simple desktop or laptop with a decent GPU and Noscribe would improve the speed. Noscribe is already providing better results compared to expensive or complicated services/applications, thank you from the cultural heritage orgs and advisors 🙃

**Comment by kaixxx** [comment]
I have compiled a test version with cuda support: https://github.com/kaixxx/noScribe/discussions/50
You can test it out if you have an NVIDIA graphics card. If the feedback is positive, I'm going to release this soon.

**Reply by VBodrov** [reply]
Tried it with GTX 1050 Ti 4Gb, have failed on «transcribe» phase with «out of memory error». I haven't had time to tinker with it yet, so I'll try again this week.

**Reply by kaixxx** [reply]
Thank you. Interesting results, let's continue the discussion here: https://github.com/kaixxx/noScribe/discussions/50#discussioncomment-8825916
